pkg/invoke/args.go

```diff
@@ -32,7 +32,7 @@ type inherited struct{}
 
 var inheritArgsFromEnv inherited
 
-func (_ *inherited) AsEnv() []string {
+func (i *inherited) AsEnv() []string {
```

Comment:
the recommendation from `staticcheck` is "omit the name if it is unused"

Reply:
Updated, thanks! @bboreham